instana/agent/host.py

```diff
@@ -157,15 +156,16 @@ def is_agent_listening(self, host, port):
         result = False
         try:
             url = "http://%s:%s/" % (host, port)
-            response = self.client.get(url, timeout=0.8)
+            response = self.client.get(url, timeout=5000)
```

Comment:
_issue:_ If I read https://requests.readthedocs.io/en/latest/user/advanced/#timeouts correctly, the timeout is specified as the number of seconds, not milliseconds. That is probably why it was 0.8 before? 5000 _seconds_ would be a really long timeout. 5 seconds would be good I guess.